scripts/run-ui-tests.sh

```diff
@@ -37,6 +37,6 @@ echo "▸ Test ${SCHEME}"
 
 xcodebuild clean test \
 	-scheme $SCHEME \
-	-destination 'platform=iOS Simulator,OS=18.2,name=iPhone 16' \
-	-sdk iphonesimulator18.2 \
+	-destination 'platform=iOS Simulator,OS=18.3,name=iPhone 16' \
+	-sdk iphonesimulator18.3 \
```

Comment:
```suggestion
	-destination 'platform=iOS Simulator,name=iPhone 16' \
```
This will use an available simulator runtime, which is good enough for now and avoids hard-coding

Reply:
Much easier, thanks!